cmd/promtool/main.go

```diff
@@ -162,6 +163,8 @@ func main() {
 		"A list of one or more files containing recording rules to be backfilled. All recording rules listed in the files will be backfilled. Alerting rules are not evaluated.",
 	).Required().ExistingFiles()
 
+	featureList := app.Flag("enable-feature", "Comma separated feature names to enable (only PromQL related). Valid options: promql-at-modifier, promql-negative-offset. See https://prometheus.io/docs/prometheus/latest/disabled_features/ for more details.").Default("").Strings()
```

Comment:
How could we keep that in sync with cmd/prometheus? Maybe we should just link to the web page?

Reply:
Sure, that sounds good